index.cjs

```diff
@@ -728,8 +728,35 @@ const config = {
   ],
 };
 
+// This class extends the built-in Error class and provides additional
+// functionality for error handling and message formatting.
+class AddLineBreaksForEslintOutputInVscodeError extends Error {
+  /**
+   * @param {string} message
+   */
+  constructor(message) {
+    super();
+    this._message = message;
+    this._firstErrorReplaced = false;
+  }
+
+  get message() {
+    return this._message;
+  }
+
+  set message(value) {
+    if (!this._firstErrorReplaced) {
+      this._message = value.replace(
+        'Error:',
+        '\n❗️Error:\nSafeQL configuration failed\n',
+      );
+      this._firstErrorReplaced = true;
+    }
+  }
```

Comment:
This setter method is triggered when assigning a new value to the message property. Inside the setter, there's a check using the `_firstErrorReplaced` variable. If it is false, it means that the replacement hasn't occurred yet. In this case, the `replace()` method is used to replace the first occurrence of the word `"Error:"`. After the replacement, `_firstErrorReplaced` is set to true to ensure that the replacement is applied only once.